src/styles/components/_accordion.scss

```diff
@@ -20,23 +20,8 @@ $dso-accordion-light-border-color: $grijs-20;
   .dso-section-handle {
     font-weight: 600;
   }
-}
 
-.dso-accordion {
   &:not(.dso-accordion-compact) {
-    .dso-accordion-section {
```

Comment:
Hier lijkt iets drastisch mis te gaan, nu wordt dit stukje verplaatst naar onder.

Reply:
Klopt, en dat is ook de bedoeling. De selector is hetzelfde, dus gaat de linter klagen

Reply:
Je hebt gelijk, het was lastig te zien en ik dacht dat het nu buiten een &:not(.dso-accordion-compact) gezet zou worden, maar dat is niet zo.